xbmc/input/MouseStat.h

```diff
@@ -32,20 +32,24 @@
 #define XBMC_BUTTON_WHEELDOWN	5
 #define XBMC_BUTTON_X1		6
 #define XBMC_BUTTON_X2		7
+#define XBMC_BUTTON_X3		8
+#define XBMC_BUTTON_X4		9
 #define XBMC_BUTTON_LMASK	XBMC_BUTTON(XBMC_BUTTON_LEFT)
 #define XBMC_BUTTON_MMASK	XBMC_BUTTON(XBMC_BUTTON_MIDDLE)
 #define XBMC_BUTTON_RMASK	XBMC_BUTTON(XBMC_BUTTON_RIGHT)
 #define XBMC_BUTTON_X1MASK	XBMC_BUTTON(XBMC_BUTTON_X1)
 #define XBMC_BUTTON_X2MASK	XBMC_BUTTON(XBMC_BUTTON_X2)
+#define XBMC_BUTTON_X3MASK	XBMC_BUTTON(XBMC_BUTTON_X3)
+#define XBMC_BUTTON_X4MASK	XBMC_BUTTON(XBMC_BUTTON_X4)
 
 #define MOUSE_MINIMUM_MOVEMENT 2
 #define MOUSE_DOUBLE_CLICK_LENGTH 500L
 #define MOUSE_ACTIVE_LENGTH   5000L
 
-#define MOUSE_MAX_BUTTON 5
+#define MOUSE_MAX_BUTTON 7
 
 enum MOUSE_STATE { MOUSE_STATE_NORMAL = 1, MOUSE_STATE_FOCUS, MOUSE_STATE_DRAG, MOUSE_STATE_CLICK };
-enum MOUSE_BUTTON { MOUSE_LEFT_BUTTON = 0, MOUSE_RIGHT_BUTTON, MOUSE_MIDDLE_BUTTON, MOUSE_EXTRA_BUTTON1, MOUSE_EXTRA_BUTTON2 };
+enum MOUSE_BUTTON { MOUSE_LEFT_BUTTON = 0, MOUSE_RIGHT_BUTTON, MOUSE_MIDDLE_BUTTON, MOUSE_EXTRA_BUTTON1, MOUSE_EXTRA_BUTTON2, MOUSE_EXTRA_BUTTON3, MOUSE_EXTRA_BUTTON4 };
```

Comment:
Please re-align the MOUSE_BUTTON enum (like MOUSE_STATE). That line is way too long ;)